integr8ly/integreatly-operator
# MGDAPI-5377 Updates to alerts check script to be more flexible

# Issue link

https://issues.redhat.com/browse/MGDAPI-5377

# What


Changes to make the script for checking the alerts more flexible
- parametrized sleep time
- added timestamp to generated csv file names
- added alert severity do the generated csv files
- updated test cases that mentioned the script

# Verification steps




Run the script
./alerts-check.sh
./alerts-check.sh 30
./alerts-check.sh 30 3scale

and see the results (generated csv files)
